docs/content/why-temporal.md

```diff
@@ -0,0 +1,150 @@
+
+import CenteredImage from "../components/CenteredImage.js"
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Temporal exists to abstract the complexity of a distributed application.
+Distributed applications exist to scale computation across multiple machines as the potential load of an application changes.
+In theory a distributed system facilitates a reliable and highly performant application.
+
+<CenteredImage
+imagePath="/diagrams/basic-distributed-system.svg"
+imageSize="75"
+title="Basic distributed system"
+/>
+
+Many modern traditional distributed systems consist of a mixture of stateless services, databases, cron jobs, and queues.
+As these systems scale, however, responding to multiple asynchronous events, communicating with unreliable external resources, or tracking the state of something very complex becomes very challenging.
+
+A large investment must be made to maintain the health of each individual component, visualize the health of the overall system, define timeout constraints for computations, and orchestrate retries for computations that fail.
+
+Temporal reconfigures the use of services, databases, cron jobs, and queues into the Temporal Platform.
+
+In a traditional system the service exists to spawn function executions.
+The Temporal Platform exists to facilitate Workflow Executions.
+
+<CenteredImage
+imagePath="/diagrams/temporal-vs-traditional.svg"
+imageSize="100"
+title="Temporal vs Traditional system"
+/>
+
+Although the two systems seem similar at first glance, they differ in several significant ways.
+
+**Failure**
+
+With a traditional system, a service function execution is both volatile and short-lived.
+
+- If a function execution fails, it's not resumable because all execution state is lost. The longer a function execution awaits, the higher the change of failure.
+- A traditional function execution typically has a limited lifespan, often measured in minutes.
+
+With Temporal, a Workflow Execution is resumable.
+
+- A Workflow Execution is fully resumable after a failure.
+- Temporal imposes no deadlines on Workflow Executions.
+
+**State**
+
+With a traditional system, stoppage or failure means that all execution state is lost.
+Your application (or a supporting component) must monitor the service's response to initiate a retry of the service execution.
+A retry starts from its _initial_ state.
+
+With Temporal, computation resumes from its _latest_ state. All progress is retained.
+
+**Communication**
+
+With a traditional system, you can't communicate with a function execution.
+
+With Temporal, Signals and Queries enable data to be sent to or extracted from a Workflow Execution.
+
+**Scope**
+
+With a traditional system, a service function execution can at best represent a business process.
+Typically, it represents only a part of a business process.
+
+A Temporal Workflow Execution can represent a business process or an entire business object.
+
+### Example subscription use-case
```

Comment:
Find-replace use-case -> "use case"? I believe the latter is much more common. https://en.wikipedia.org/wiki/Use_case